webapp/src/main/java/com/box/l10n/mojito/service/scheduledjob/ScheduledJobTriggerListener.java

```diff
@@ -0,0 +1,49 @@
+package com.box.l10n.mojito.service.scheduledjob;
+
+import com.box.l10n.mojito.entity.ScheduledJob;
+import java.util.Optional;
+import org.quartz.JobExecutionContext;
+import org.quartz.Trigger;
+import org.quartz.listeners.TriggerListenerSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Listener that receives events about jobs that are about to be triggered under the 'scheduledJobs'
```

Comment:
Exactly! The job will never run because of the veto function.

You are right about if we want to support this scenario, we would have to delete the schedule and then create it again just to pause a job. I did try using the pause method that Quartz has to offer for jobs which would pause the job but strangely enough when I would resume the job the schedule would be off and sometimes the job wouldn't even trigger in certain cases. I went with the approach of using the trigger as the decider on if a job should run or not which works very well as the schedule is always correct!

Reply:
Thanks for confirming that!